Issue when running the gradlew build

I've successfully compiled the engine with scons with the admob module included.

When running the `./gradlew build` command inside the `/platform/android/java` directory, I'm getting the error back:
`
> Task :compileDebugJavaWithJavac
/home/user/godot/godot-master/modules/admob/android/GodotAdMob.java:67: error: <anonymous org.godotengine.godot.GodotAdMob$1$1> is not abstract and does not override abstract method onRewardedVideoCompleted() in RewardedVideoAdListener
                                {
                                ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
FAILURE: Build failed with an exception.
What went wrong:
Execution failed for task ':compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
`

I managed to build gradle successfully when I had no third party/custom modules inserted into the core.

Have I missed something really simple? I've made sure the dependencies are all as per the guide.

Any help would be great, thanks.

--- a/admob/android/GodotAdMob.java
+++ b/admob/android/GodotAdMob.java
@@ -107,6 +107,12 @@ public class GodotAdMob extends Godot.SingletonBase
 						Log.w("godot", "AdMob: onRewardedVideoStarted");
 						GodotLib.calldeferred(instance_id, "_on_rewarded_video_started", new Object[] { });
 					}
+
+					@Override
+					public void onRewardedVideoCompleted() {
+						Log.w("godot", "AdMob: onRewardedVideoCompleted");
+						GodotLib.calldeferred(instance_id, "_on_rewarded_video_completed", new Object[] { });
+					}
 				});
 
 			}

--- a/admob/ios/src/AdmobRewarded.mm
+++ b/admob/ios/src/AdmobRewarded.mm
@@ -94,4 +94,11 @@
     obj->call_deferred("_on_rewarded_video_ad_failed_to_load", (int)error.code);
 }
 
+- (void)rewardBasedVideoAdDidCompletePlaying:(GADRewardBasedVideoAd *)rewardBasedVideoAd {
+    NSLog(@"Reward based video ad has completed.");
+    Object *obj = ObjectDB::get_instance(instanceId);
+    obj->call_deferred("_on_rewarded_video_completed");
+}
+
+
 @end

